<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlotterFilterPane</class>
 <widget class="QWidget" name="BlotterFilterPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlotterFilterPane</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinPrice">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinSize">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnApply</sender>
   <signal>clicked(bool)</signal>
   <receiver>BlotterFilterPane</receiver>
   <slot>OnApplyCliecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnApplyCliecked(bool)</slot>
 </slots>
</ui>
